source/module_hamilt_lcao/module_gint/grid_meshball.cpp

```diff
@@ -127,7 +128,7 @@ double Grid_MeshBall::deal_with_atom_spillage(const double *pos)
 					cell[ip] = i*this->bigcell_vec1[ip] +
 						j*this->bigcell_vec2[ip] +
 						k*this->bigcell_vec3[ip];
-					dx += std::pow(cell[ip] - pos[ip], 2);
+					dx += pow(cell[ip] - pos[ip], 2);
```

Comment:
I would suggest using another name than 'pow' to avoid confusion. For example, pow_int